samples/unity-of-bugs/Assets/Scripts/BuildtimeOptionsConfiguration.cs

```diff
@@ -1,14 +1,16 @@
 using UnityEngine;
 using Sentry.Unity;
 
-[CreateAssetMenu(fileName = "Assets/Resources/Sentry/BuildtimeOptionsConfiguration.asset", menuName = "Sentry/BuildtimeOptionsConfiguration.asset", order = 999)]
-public class BuildtimeOptionsConfiguration : SentryBuildtimeOptionsConfiguration
+[CreateAssetMenu(fileName = "Assets/Resources/Sentry/BuildTimeOptionsConfiguration.asset", menuName = "Sentry/BuildTimeOptionsConfiguration", order = 999)]
+public class BuildTimeOptionsConfiguration : Sentry.Unity.SentryBuildTimeOptionsConfiguration
 {
-    /// See base class for documentation.
+    /// Called during app build. Changes made here will affect build-time processing, symbol upload, etc.
+    /// Additionally, because iOS, macOS and Android native error handling is configured at build time,
+    /// you can make changes to these options here.
     /// Learn more at https://docs.sentry.io/platforms/unity/configuration/options/#programmatic-configuration
     public override void Configure(SentryUnityOptions options, SentryCliOptions cliOptions)
     {
-        Debug.Log("SentryBuildtimeOptionsConfiguration::Configure() called");
-        Debug.Log("SentryBuildtimeOptionsConfiguration::Configure() finished");
+        Debug.Log("SentryBuildTimeOptionsConfiguration::Configure() called");
```

Comment:
you can avoid trying to keep names in sync manually with `nameof`:

```suggestion
        Debug.Log(nameof(SentryBuildTimeOptionsConfiguration) + "::Configure() called");
```